cpp/tests/reductions/segmented_reduction_tests.cpp

```diff
@@ -15,10 +15,12 @@
  */
 
 #include <cudf_test/base_fixture.hpp>
+#include <cudf_test/column_utilities.hpp>
```

Comment:
@isVoid Can you revert this file? (It's meant to be part of another PR.)

Reply:
Yeah, just about to do that.